app/models/book.rb

```diff
@@ -33,6 +33,18 @@ def self.to_csv
     end
   end
 
+  def leap_year?
+    year = created_at.year
+
+    if year % 4 == 0 && year % 100 != 0
+      true
+    elsif year % 400 == 0
+      true
+    else
+      false
+    end
```

Comment:
shorthand for this is:

```ruby
(year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)
```

There are times when shortening an if/elsif statement will be more readable, and times when it makes it harder to read.

It's going to be up to you to determine when to use what you have, and when to use something like what I've written here.

Reply:
I think the branching logic is easier for editing, but this is more concise. I'll go with your code since the concept of a leap year is a solid concept that exists outside of this app and we can reasonably assume won't change.

Reply:
Taking it another step forward, we could make it read like english:

```ruby
divisible_by_4 = year % 4 == 0
not_divisible_by_100 = year % 100 != 0
divisible_by_400 = year % 400 == 0

(divisible_by_4 && not_divisible_by_100) || divisible_by_400
```

Again - up to you which you find more readable.